game.js

```diff
@@ -0,0 +1,143 @@
+var sizeOfOneSquare = 40, sizeOfFoundation = 3;
+var canvas = document.getElementById("main"), context = 
+canvas.getContext("2d");
+function drawmap() {
+    for (var i=0; i<sizeOfFoundation; i++){
```

Comment:
create variables before for loop

Reply:
done, but why? 
code is more simple like I wrote ,no?

Reply:
not